src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -136,11 +136,25 @@ public void Analyze_CriticalException_ExceptionNotCaught()
         }
 
         [TestMethod]
-        public void Dispose_DisposesHttpWrapper()
+        public void Dispose_EslintBridgeProcessIsNotRunning_DisposesHttpWrapper()
```

Comment:
Could add a comment saying this is a regression test for https://github.com/SonarSource/sonarlint-visualstudio/issues/2469